src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {AaveV3PayloadArbitrum} from 'aave-helpers/v3-config-engine/AaveV3PayloadArbitrum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+/**
+ * @title  Adjusting Interest Rate Curve for weETH on Arbitrum
+ * @author ACI
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xed2fd3dfee1f29f04b6cda4a5c4629fcca32a5c961b1b3e2a49ba6842367ce31
+ * - Discussion: https://governance.aave.com/t/arfc-adjusting-interest-rate-curve-for-weeth-on-arbitrum/17804
+ */
+contract AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603 is
+  AaveV3PayloadArbitrum
+{
+  function rateStrategiesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.RateStrategyUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.RateStrategyUpdate[]
+      memory rateStrategies = new IAaveV3ConfigEngine.RateStrategyUpdate[](1);
+    rateStrategies[0] = IAaveV3ConfigEngine.RateStrategyUpdate({
+      asset: AaveV3ArbitrumAssets.weETH_UNDERLYING,
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(35_00),
+        baseVariableBorrowRate: _bpsToRay(0),
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(300_00),
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(0),
+        stableRateExcessOffset: _bpsToRay(0),
+        optimalStableToTotalDebtRatio: _bpsToRay(0)
```

Comment:
What is the reason for changing the stable interest rates? 
It should not make any difference no as stable borrowing is anyway disabled and being deprecated. 

If the reason is to align stable rates across all network for weETH, even doing so there will be inconsistency as stable rate slope 1 and stable rate slope 2 for weETH on mainnet is 0%.

Reference: [Dashboard Link](https://dash.onaave.com/#chainId=&type=&symbol=weETH&fields=symbol%2CbaseStableBorrowRate%2CbaseVariableBorrowRate%2CstableRateSlope1%2CstableRateSlope2%2CvariableRateSlope1%2CvariableRateSlope2%2CstableBorrowRateEnabled%2CborrowableInIsolation%2CisSiloedBorrowing)

Reply:
you're correct we do not want to change the stable rate. KEEP_CURRENT was initial intent